smrf/distribute/precipitation.py

```diff
@@ -475,70 +490,24 @@ def distribute_thread(self, queue, data, date, mask=None):
 
             # variables for wind redistribution
             az = queue['wind_direction'].get(t)
-            wind_speed = queue['wind_speed'].get(t)
             flatwind = queue['flatwind'].get(t)
             dir_round_cell = queue['dir_round_cell'].get(t)
             cell_maxus = queue['cellmaxus'].get(t)
 
             self.distribute(data.precip.loc[t], dpt, precip_temp, ta, t,
                             data.wind_speed.loc[t], data.air_temp.loc[t],
-                            az, dir_round_cell, flatwind, cell_maxus, mask=mask)
+                            az, dir_round_cell, flatwind, cell_maxus,
+                            mask=mask)
 
             queue[self.variable].put([t, self.precip])
-
             queue['percent_snow'].put([t, self.percent_snow])
-
             queue['snow_density'].put([t, self.snow_density])
-
             queue['last_storm_day_basin'].put([t, self.last_storm_day_basin])
-
             queue['storm_days'].put([t, self.storm_days])
-
             queue['storm_total'].put([t, self.storm_total])
             if self.nasde_model == "marks2017":
                 queue['storm_id'].put([t, self.storm_id])
 
     def post_processor(self, main_obj, threaded=False):
 
         pass
```

Comment:
Does this method need to exist or can't it be removed completely?

Reply:
I don't remember why the post processor is there. I think we were working on a possible method to process the data after it was generated but it never materialized. Probably should remove. #172